src/NzbDrone.Core/Notifications/Webhook/WebhookMovie.cs

```diff
@@ -28,6 +29,7 @@ public WebhookMovie(Movie movie)
             FolderPath = movie.Path;
             TmdbId = movie.TmdbId;
             ImdbId = movie.ImdbId;
+            Overview = movie.Overview;
```

Comment:
Needs fixed, no Overview directly on movie

Reply:
Would it be okay to add Overview to the objects in https://github.com/Radarr/Radarr/blob/develop/src/NzbDrone.Core/Movies/Movie.cs or is there another way to handle this?

Reply:
The overview is in the MovieMetadata class which is already included in the Movie class so movie.MovieMetadata.value.Overview should do it (casing is likely incorrect as I am on mobile)